deploy-azure-containerapp-v1.yaml

```diff
@@ -0,0 +1,306 @@
+on:
+  workflow_call:
+    inputs:
+      dockerfile_path:
+        type: string
+        required: false
+        default: './Dockerfile'
+        description: Dockerfile path
+      dockerfile_context:
+        type: string
+        required: false
+        default: '.'
+        description: Dockerfile build context
+      docker_image_name:
+        type: string
+        required: false
+        default: ${{ github.repository }}
+        description: Docker image name, defaults to repository name
+      docker_image_description:
+        type: string
+        required: true
+        description: A description to use as image label
+      docker_image_authors:
+        type: string
+        required: false
+        default: PagoPA
+        description: Authors names to use as image label
+      build_args:
+        description: List of build arguments to use for Dockerfile build, given in env=value format.
+        type: string
+        required: false
+      build_platforms:
+        type: string
+        required: false
+        default: linux/amd64
+        description: Image runtime platform, supports multiple values
```

Comment:
question: what does it mean "multiple values"? comma separated list of... ?

Reply:
I specified to use a comma separated list as [required](https://docs.docker.com/reference/cli/docker/buildx/build/#platform)